cloudbuild/run-tests.sh

```diff
@@ -101,9 +108,16 @@ main() {
   cd /init-actions
   configure_gcloud
   configure_gcloud_ssh_key
-  initialize_git_repo
-  determine_tests_to_run
-  run_tests
+  if [[ $RUN_ALL_TESTS=="true" ]]; then
+    # Run periodic
+    initialize_git_repo "false"
```

Comment:
Do we even need to call this function here?

Reply:
I'm calling it so that we can call "git rev-parse head" to get the commit number. Maybe, there is a better way?

Reply:
Do you need `git rev-parse head` for TestGrid?

Reply:
Not really. I was getting it hoping that I could have it surface in TestGrid so that we can know which test run column corresponds to which code version. However, I'm already providing the GCS build number which surfaces the commit num. Example: https://screenshot.googleplex.com/ZRpNKg4CZv8.png